scripts/deployment/deploy_pg.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import dataclasses
+import argparse
+import logging
+import os
+import psycopg2
+import copy
+
+
+FUNCTION_FILES = ".sql"
+DDL_FILES = ".ddl"
+DB_CREATION_FILES = "00_"
+SCHEMA_CREATION_FILE = "_" + DDL_FILES
+
+
+
+@dataclasses.dataclass
+class PostgresDBConn:
+    """This dataclass contains all information related to making a connection to Postgres DB."""
+    username: str
+    password: str
+    host: str
+    database: str
+    port: int
+
+
+def parse_args() -> argparse.Namespace:
+    """CLI args parsing function."""
+    parser = argparse.ArgumentParser(
+        description="Deploys structures to DB. (script version: 1.0)",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    parser.add_argument(
+        "-ph", "--host",
+        help="database server host (default: \"localhost\")",
+        default="localhost",
+    )
+    parser.add_argument(
+        "-p", "--port",
+        help="database server port (default: \"5432\")",
+        default="5432",
```

Comment:
```suggestion
        default=5432,
```

perhaps it would be converted automatically anyway though. But Python is a magical place where anything is allowed and that makes software very vulnerable to mistakes